src/ecc/curves.iced

```diff
@@ -208,6 +219,18 @@ exports.Curve25519 = class Curve25519 extends Curve
 
   #----------------------------------
 
+  # Serialize single curve coordinate (usually private key X) to MPI
+  # buffer, with 2-byte header encoding bit-size of the coordinate.
+  coord_to_mpi_buffer : (p) ->
+    byte_size = @mpi_coord_byte_size()
+    throw new Error "Invalid p.length: got #{p.length} expected #{byte_size}." if p.length != byte_size
```

Comment:
I think the right idea is to throw when the key is generated or imported if this error exists. And, as you mention in Keybase chat, to `throw` properly in `@_alloc`, which might be where the real bug is.